deepchem/dock/tests/test_pose_scoring.py

```diff
@@ -26,34 +25,51 @@ class TestPoseScoring(unittest.TestCase):
   Does sanity checks on pose generation.
   """
 
-  def setUp(self):
-    """Downloads dataset."""
-    download_url(
-        "http://deepchem.io.s3-website-us-west-1.amazonaws.com/featurized_datasets/core_grid.json"
-    )
-    json_fname = os.path.join(get_data_dir(), 'core_grid.json')
-    self.core_dataset = dc.data.NumpyDataset.from_json(json_fname)
-
-  def test_pose_scorer_init(self):
-    """Tests that pose-score works."""
-    sklearn_model = RandomForestRegressor(n_estimators=10)
-    model = dc.models.SklearnModel(sklearn_model)
-    logger.info("About to fit model on core set")
-    model.fit(self.core_dataset)
-
-    pose_scorer = dc.dock.GridPoseScorer(model, feat="grid")
-
-  def test_pose_scorer_score(self):
-    """Tests that scores are generated"""
-    current_dir = os.path.dirname(os.path.realpath(__file__))
-    protein_file = os.path.join(current_dir, "1jld_protein.pdb")
-    ligand_file = os.path.join(current_dir, "1jld_ligand.sdf")
-
-    sklearn_model = RandomForestRegressor(n_estimators=10)
-    model = dc.models.SklearnModel(sklearn_model)
-    logger.info("About to fit model on core set")
-    model.fit(self.core_dataset)
-
-    pose_scorer = dc.dock.GridPoseScorer(model, feat="grid")
-    score = pose_scorer.score(protein_file, ligand_file)
-    assert score.shape == (1,)
+  def test_cutoff_filter(self):
+    N = 10
+    M = 5
+    d = np.ones((N, M))
+    x = np.random.rand(N, M)
+    cutoff_dist = 0.5
+    x_thres = cutoff_filter(d, x, cutoff=cutoff_dist)
+    assert (x_thres == np.zeros((N, M))).all()
+
+  def test_vina_nonlinearity(self):
+    N = 10
+    M = 5
+    c = np.random.rand(N, M)
+    Nrot = 5
+    w = 0.5
+    out_tensor = vina_nonlinearity(c, w, Nrot)
+    assert out_tensor.shape == (N, M)
```

Comment:
Just checking the shape isn't much of a test.  Can you verify that the result is actually correct?

Reply:
Added better tests here.